patches/net-cookies-canonical_cookie.cc.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
+index 91611ac4171c19a031044ae6b1459acce246d427..daf61181bcaebef8b53f9e5de43319cc0a26e698 100644
+--- a/net/cookies/canonical_cookie.cc
++++ b/net/cookies/canonical_cookie.cc
+@@ -89,6 +89,11 @@ int PartialCookieOrdering(const CanonicalCookie& a, const CanonicalCookie& b) {
+ 
+ }  // namespace
+ 
++const TimeDelta CanonicalCookie::kMaxClientSideExpiration =
```

Comment:
could you put this into `brave/chromium_src/net/cookies/canonical_cookie.cc` and put it into an anonymous namespace to reduce patch?
Also there is a `TimeDelta::FromDays` that you can use.